frontend/src/views/MyProgress/information/QualityMeasureInfo.vue

```diff
@@ -1,11 +1,59 @@
 <template>
-  <div>
-    Appro information
+  <div class="fr-text">
+    <p>
+      Depuis 2022, les repas doivent comporter au moins {{ applicableRules.qualityThreshold }} % de produits de qualité
+      et durables dont au moins {{ applicableRules.bioThreshold }} % issus de l’agriculture biologique ou en conversion,
+      pour les cantines {{ regionDisplayName ? `dans la région de ${regionDisplayName}` : "en France métropolitaine" }}.
```

Comment:
Pour éviter des soucis avec les régions qui commencent par une voyelle et donc devraient être affichés avec `d'` au lieu de `de`.
```suggestion
      pour les cantines {{ regionDisplayName ? `dans la région « ${regionDisplayName} »` : "en France métropolitaine" }}.
```

Reply:
https://github.com/betagouv/ma-cantine/pull/3100/commits/aa5d57b4566cc2c7c0d3b1325f36317a940c81a5